Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -28,7 +28,7 @@ describe("Vaccine Card Page", () => {
         cy.get("[data-testid=vaccineCardFormTitle]").should("be.visible");
         cy.get("[data-testid=btnCancel]").should("be.visible");
         cy.get("[data-testid=btnEnter]").should("be.enabled", "be.visible");
-        cy.get("[data-testid=btnPrivacyStatement]").should("be.visible");
+        cy.get("[data-testid=privacy-statement-button]").should("be.visible");
```

Comment:
```suggestion
        cy.get("[data-testid=btnPrivacyStatement]").should("be.visible");
```

I've branched off a new folder for vuetify tests, so we don't need to worry about being backwards compatible which makes this change unnecessary.